src/main/java/org/reactome/release/qa/check/CoV2InfectionPathwayEventCheck.java

```diff
@@ -0,0 +1,176 @@
+            // issue is being reported, it indicates they likely made an error.
+            // If literatureReference attribute does not contain at least 1 litRef from 2020.
+            if (cov2Event.getAttributeValue(ReactomeJavaConstants.inferredFrom) == null && !hasRecentLiteratureReference(cov2Event)) {
+                issues.add("Does not contain a 2020 literature reference");
+            }
+            // Summation text was updated near end of release cycle where CoV-1-to-CoV-2 projections happened.
+            // If it didn't have the updated text, there are two reasons for it, outlined below.
+            if (!hasCorrectSummationFormat(cov2Event)) {
+                if (cov2Event.getAttributeValue(ReactomeJavaConstants.inferredFrom) == null) {
+                    issues.add("InferredFrom is empty but instance has COVID inference summation text");
+                } else {
+                    issues.add("InferredFrom is populated but instance does NOT have COVID inference summation text.");
+                }
+            }
+            if (issues.size() > 0) {
+                report.addLine(getReportLine(cov2Event, String.join("|",issues)));
+            }
+        }
+        report.setColumnHeaders(getColumnHeaders());
+        return report;
+    }
+
+    /**
+     * Checks if instance created dateTime is before most recent modified instance's dateTime. Also checks if
+     * any modified instance is present -- it should be given that there was at least 1 required modification from the
+     * CoV-1-to-CoV-2 merge.
+     * @param cov2Event - GKInstance, Event contained within 'SARS-CoV-2 Infection' Pathway.
+     * @return - boolean, true if modification property looks correct, false if not.
+     * @throws Exception, thrown by MySQLAdaptor.
+     */
+    private boolean hasRecentlyModifiedSummation(GKInstance cov2Event) throws Exception {
+       for (GKInstance summation : (Collection<GKInstance>) cov2Event.getAttributeValuesList(ReactomeJavaConstants.summation)) {
+            GKInstance createdInst = (GKInstance) summation.getAttributeValue(ReactomeJavaConstants.created);
+            List<GKInstance> modifieds = summation.getAttributeValuesList(ReactomeJavaConstants.modified);
+            if (modifieds.size() > 0) {
+                // At time of writing (August 2020), nothing is reported from this clause, but it is a good check.
+                GKInstance mostRecentModifiedInst = modifieds.get(modifieds.size() - 1);
+                // Only check Year/Month/Day since the changes took place more than a day before Curators got to look at them in gk_central.
+                int createdDateTime = Integer.valueOf(createdInst.getAttributeValue(ReactomeJavaConstants.dateTime).toString().split(" ")[0].replaceAll("-", ""));
+                int modifiedDateTime = Integer.valueOf(mostRecentModifiedInst.getAttributeValue(ReactomeJavaConstants.dateTime).toString().split(" ")[0].replaceAll("-", ""));
```

Comment:
This is a bit of a mouthful.  Could this be a method like `getDate(GKInstance instanceEdit)` (maybe something for release-common-lib, actually).  

Also, it's a bit unusual to have a date represented as a raw `int`.  I won't insist on changing it here, but you could use the Java built-in `LocalDate`: https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html.

It has methods for comparing dates like `isAfter(otherDate)` and `isBefore(otherDate)` and returning a boolean.

Reply:
I don't think a special function in release-common-lib is necessary, but I do agree that it's best to compare dates using libraries that were specifically written for that purpose.

Reply:
I was able to get them as a `LocalDate` object and use the `isAfter()` method with the same results